### PR TITLE
fix(color palette): added color palette for CRDT county charts

### PR DESCRIPTION
**Pull request by goldfarb**
## Description

* added 6 color, colorblind safe color palette
* included all possible census categories for the county chart legend and chart
* mapped color palette to the census categories

### CODE REVIEW

**Comment by goldfarb** [comment]
Two options for color mapping, with the current charts: 
[A](https://github.com/COVID19Tracking/website/pull/993/commits/58dee748b6322a39eea91b589b80636a337b9226)  

image 
image

or 

[B](https://github.com/COVID19Tracking/website/pull/993/commits/24dc4fc98219cb8131594640c4a3f146549a69fb)

image
image

or

*other*

**Comment by tealtan** [comment]
Thanks! With option a, the dark green and blue seem a bit close to me, so I’m leaning towards option b.

File: src/scss/colors.module.scss [hunk 1]
```diff
@@ -66,6 +66,17 @@ $color-infobox-alert-dark: #802509;
 $color-error-light: #f9ded9;
 $color-error-dark: #e35942;
 
+
+// Six Color Palette
```
Comment (schwartzadev) [inline src/scss/colors.module.scss:70]: Could you please add a prefix to indicate the usage of this palette? Maybe something like `county-` for `color-county-palette-100`?

File: src/components/pages/race/charts/county-chart.module.scss [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+.aian-alone {
+  fill: $color-palette-500;
+}
+.asian-alone {
+  fill: $color-palette-600;
+}
+
+.two-or-more {
+  fill: red;
 }
```
Comment (schwartzadev) [inline src/components/pages/race/charts/county-chart.module.scss:25]: Based on the legend SASS file, should `.two-or-more` have `fill: #444;`?

File: src/scss/colors.module.scss (hunk 1 above)
Comment (goldfarb) [reply src/scss/colors.module.scss:70]: I think it is for use outside of this chart. I could call it `qualitative-color-palette`?

**Comment by goldfarb** [comment]
@schwartzadev One final rearrangement, an [option C (054ce20)](https://github.com/COVID19Tracking/website/pull/993/commits/054ce2016a0cb3bac2689d36918b381aade5be65)

where the color arrangement is hopefully a little better and also where I noticed that the `two-or-more` colors weren't the same, so that has been updated as well

image

File: src/components/pages/race/charts/county-chart.module.scss (hunk 2 above)
Comment (goldfarb) [reply src/components/pages/race/charts/county-chart.module.scss:25]: Yes! I noticed that and changed both to `#555`

 [(054ce20)](https://github.com/COVID19Tracking/website/pull/993/commits/054ce2016a0cb3bac2689d36918b381aade5be65)

File: src/scss/colors.module.scss (hunk 1 above)
Comment (schwartzadev) [reply src/scss/colors.module.scss:70]: Ah, got it! That would be great -- just something a little more descriptive 👍

**Comment by schwartzadev** [comment]
Thanks! Re: colors, I'll defer to @tealtan, but that looks good to me!

File: src/scss/colors.module.scss (hunk 1 above)
Comment (goldfarb) [reply src/scss/colors.module.scss]: Renamed! [35eed6a](https://github.com/COVID19Tracking/website/pull/993/commits/35eed6ac70c423b877e29e080d2f1427051174a7)

File: src/scss/colors.module.scss [hunk 3]
```diff
@@ -13,7 +13,6 @@ $color-plum-200: #d1d1e8;
 $color-plum-300: #b6b7db;
 $color-plum-400: #8b8dc7;
 $color-plum-500: #6164ba;
-$color-plum-550: #5667ad;
```
Comment (schwartzadev) [inline src/scss/colors.module.scss:16]: It looks like you accidentally dropped this line!

Comment (goldfarb) [reply src/scss/colors.module.scss]: Oops - it's [back](https://github.com/COVID19Tracking/website/pull/993/commits/8d6bfe383922759fe1f7d5f00e6ec28bf2206c6d)

**Comment by kevee** [comment]
Updating branch and merging. Thanks, @goldfarb!

**Pull request merged**